tests/containers/bci_prepare.pm

```diff
@@ -106,6 +106,14 @@ sub run {
         record_info('IMAGE', $image);
         script_retry("$engine pull $image", timeout => 300, delay => 60, retry => 3);
         record_info('Inspect', script_output("$engine inspect $image"));
+
+        my $build = get_var('CONTAINER_IMAGE_BUILD');
+        if ($build && $build ne 'UNKNOWN') {
+            record_info('CONTAINER_IMAGE_BUILD', $build);
+            my $reference = script_output(qq($engine inspect --type image $image | jq -r '.[0].Config.Labels."org.opensuse.reference"'));
+            record_info('org.opensuse.reference', $reference);
+            die('Miss match in image build number. The image build number is different than the one triggered by the container bot!') if ($reference !~ /$build$/);
```

Comment:
if we skip the first `record_info`, we could do:
```suggestion
            die("Miss match in image build number. The image build number extracted from the image ($reference) is different than the one triggered by the container bot ($build)!") if ($reference !~ /$build$/);
```

Reply:
Please not. I also want to see the image ref and the build number when this check is passing.